requirements.txt

```diff
@@ -3,8 +3,18 @@ matplotlib>=3.2.1
 numpy>=1.17.2
 pandas>=0.25.1
 scikit-learn>=0.22.1
+
+
+math
```

Comment:
These seems to be part of the standard library
https://docs.python.org/3/library/
except for seaborn (the only truly new one here) and scipy.stats (which is part of scipy)